atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/TokenRangeResolutionTest.java

```diff
@@ -131,6 +139,49 @@ public void startEndTokenOrderingIsSignificant() {
                 .isFalse();
     }
 
+    @Test
+    public void mediumClusterEndpointsShuffle() {
+        for (int n = 1; n < 10; n++) {
+            int nodeCount = n * 3;
+            ImmutableSortedSet<TokenRange> baseRing = createTokenRangesForNodes(nodeCount);
```

Comment:
nit: think it will be okay to use the interface here instead of explicit immutable